Libraries/Opc.Ua.Configuration/ApplicationConfigurationBuilder.cs

```diff
@@ -141,20 +143,87 @@ public IApplicationConfigurationBuilderClientSelected AsClient()
                     StoreType = rejectedRootType,
                     StorePath = DefaultCertificateStorePath(TrustlistType.Rejected, rejectedRoot)
                 },
+                // ensure secure default settings
+                AutoAcceptUntrustedCertificates = false,
+                AddAppCertToTrustedStore = false,
+                RejectSHA1SignedCertificates = true,
+                RejectUnknownRevocationStatus = true,
+                SuppressNonceValidationErrors = false,
+                SendCertificateChain = true,
+                MinimumCertificateKeySize = CertificateFactory.DefaultKeySize,
+                MinimumECCertificateKeySize = CertificateFactory.DefaultECCKeySize,
```

Comment:
I wonder if this setting makes sense. ECC key size is strictly defined by the sec profile.